### ISSUE TITLE
Different format for Last Online date on server dashboard

### ISSUE THREAD

**Issue by farwl** [Minehut/Meta#112]
### Feature request

**Platform**
 website

**Describe the feature**

Option to make this say something like 10 days ago instead of this formatted date.
![](https://cdn.discordapp.com/attachments/592130982451216402/725901073801412609/unknown.png)

**How the feature is useful**

QOL

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by iGalaxyYT** [comment]
I don't think the current formatting needs changes. How would this be beneficial?

**Comment by drtshock** [comment]
what do you mean an option? Should we make this something you set? Currently, it's localized by your browser.

**Comment by farwl** [comment]
Yes I like that its localized by your browser but a different display option would be nice. Instead of a formatted date, something like **10 days ago** would be easier to see and more user friendly imo.
